local-harnesses/build.config.json.template

```diff
@@ -23,5 +23,5 @@
         "last_name": "Dev",
         "expire_now": false
     },
-    "traversal_memory_limit": 1
+    "traversal_memory_limit": 2
```

Comment:
I would like us to remove this piece from the template. In dev, we should probably strive to use application defaults wherever possible. If this entry exists in `integration.config.json.template`, yank it out of there too